assets/src/BlockEditorValidation.js

```diff
@@ -63,46 +62,46 @@ export const blockEditorValidation = () => {
   );
 };
 
-document.addEventListener('change', e=> {
-  if (!e.target.matches('select[name="p4_campaign_name"]')) {
+document.addEventListener( 'change', ( e ) => {
```

Comment:
This is not part of this scope, but we should standardize this parameter name. For example, sometimes we use `e` and sometimes we use `evt`.